# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mapped_hyph/src/bin/hyf_compile.rs
# Commit: a1ab26abf3eb
# Full Hash: a1ab26abf3eb645e202d69dffea86398a0ea4563
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-09-30 21:45:29
# Regressor Bug: 1667192
# File Overlap Count: 6
# Description:
#   Bug 1667192 - patch 1.1 - Run `mach vendor rust` to update in-tree mapped_hyph source. r=heycam
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D91409
# ==============================================================================

diff -r 66b86f8b6682 -r a1ab26abf3eb third_party/rust/mapped_hyph/src/bin/hyf_compile.rs
--- a/third_party/rust/mapped_hyph/src/bin/hyf_compile.rs	Wed Sep 30 12:51:17 2020 +0000
+++ b/third_party/rust/mapped_hyph/src/bin/hyf_compile.rs	Wed Sep 30 12:51:54 2020 +0000
@@ -1,4 +1,4 @@
-// Copyright 2019 Mozilla Foundation. See the COPYRIGHT
+// Copyright 2019-2020 Mozilla Foundation. See the COPYRIGHT
 // file at the top-level directory of this distribution.
 //
 // Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
@@ -17,7 +17,7 @@
     if args.len() == 3 {
         let in_file = File::open(&args[1])?;
         let mut out_file = File::create(&args[2])?;
-        mapped_hyph::builder::write_hyf_file(&mut out_file, mapped_hyph::builder::read_dic_file(&in_file))?;
+        mapped_hyph::builder::compile(&in_file, &mut out_file, true)?;
     } else {
         println!("usage: hyf_compile <pattern-file> <output-file>");
     }